cve_bin_tool/cli.py

```diff
@@ -200,6 +202,8 @@ def scan_file(self, filename):
                         )
 
                     found_cves = self.get_cves(vendor_package_pairs, version)
+                    vendor = vendor_package_pairs[0][0]
+                    package = vendor_package_pairs[0][1]
```

Comment:
What happens here if there's more than one vendor package pair?  (maybe try it against a kerberos test case to find out if you don't know)